terraform-provider-jans/docs/data-sources/agama_repository.md

```
@@ -0,0 +1,40 @@
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Read-Only
+
+- `id` (String) The ID of this resource.
+- `repositories` (List of Object) List of Agama repositories (see [below for nested schema](#nestedatt--repositories))
+
+<a id="nestedatt--repositories"></a>
+### Nested Schema for `repositories`
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)